aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -166,13 +166,13 @@ class QConvPackWeightInt8 final : public c10::OperatorKernel {
     const uint32_t kernel_w = weight.size(2);
     const size_t in_ch = weight.size(3) * groups;
 
-    Tensor bias;
+    Tensor bias_fp32;
     if (bias_in.has_value()) {
-      bias = bias_in.value();
+      bias_fp32 = bias_in.value();
     } else {
-      bias = at::empty(out_ch, at::kFloat);
-      bias = at::quantize_linear(bias, 1.0, 0, kQInt32);
+      bias_fp32 = at::zeros(out_ch, at::kFloat);
     }
+    auto bias = at::quantize_linear(bias_fp32, 1.0, 0, kQInt32);
```

Comment:
I think you don't need this quantization at all - it's never used below

Reply:
It is passed to the pre-pack function since qnnpack expects quantized bias.

Reply:
Correct, bias should now be stored as float32

Reply:
Yes, it is stored in the pack struct as float32, but we still need to quantize it to call the pack function for qnnpack.